src/clients/box_ui/lib/src/widgets/large_merchants_item.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:box_ui/box_ui.dart';
+import 'package:box_ui/src/shared/styles.dart';
+import 'package:flutter/material.dart';
+
+import 'large_merchants_item_images_carousel.dart';
+
+class LargeMerchantItem extends StatelessWidget {
+  final List<String> images;
+  final String name;
+  final List<String> categories;
+  final double? rating;
+  final int? numberOfRatings;
+  final int? deliveryInMinutes;
+  final double? deliveryCost;
+  final bool isClosed;
+
+  const LargeMerchantItem(
+      {Key? key,
+      required this.images,
+      required this.name,
+      required this.categories,
+      this.deliveryInMinutes,
+      this.deliveryCost,
+      this.rating,
+      this.numberOfRatings,
+      this.isClosed = false})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        LargeMerchantsItemImagesCarsouel(
+          images: images,
+        ),
+        verticalSpaceSmall,
+        Text(name, style: subheadingStyle.copyWith(color: kcDeepGreyColor)),
+        verticalSpaceTiny,
+        Text(categories.join(' • '),
+            style: bodyStyle.copyWith(color: kcMediumGreyColor)),
+        verticalSpaceSmall,
+        Row(
+          children: [
+            if (rating != null) ...[
+              Text(
```

Comment:
We need to use the BoxText for all text in the app. That's why we built it. We shouldn't export the styles. Check out the tutorial before this, I updated it to use the BoxText.

Reply:
Sorry about that, I replaced all the styles with BoxText